hw2/Index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>hw2</title>
+	<link rel="stylesheet" href="css/reset.css">
+	<link rel="stylesheet" href="css/fonts.css">
+	<link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+	<div class="wrapper">
+		<header>
+			<img src="img/logo.png">
+			<ul class="top-menu">
+				<li><a class="top-menu_item" href="#">Главная</a></li>
+				<li><a class="top-menu_item" href="#">Товары</a></li>
+				<li><a class="top-menu_item" href="#">О_Нас</a></li>
+				<li><a class="top-menu_item" href="#">Контакты</a></li>
+			</ul>	
+		</header>
+		<section class="container">
+			<aside class="left-menu">
+				<ul>
+					<li class="left-menu_item"><a class="left-menu_link" href="#">Ноутбуки</a></li>
+					<li class="left-menu_item"><a class="left-menu_link" href="#">Планшеты</a></li>
+					<li class="left-menu_item"><a class="left-menu_link" href="#">Телефоны</a></li>
+					<li class="left-menu_item"><a class="left-menu_link" href="#">Телевизоры</a></li>
+					<li class="left-menu_item"><a class="left-menu_link" href="#">Бытовая техника</a></li>
+					<li class="left-menu_item"><a class="left-menu_link" href="#">Автотовары</a></li>
+				</ul>
+			</aside>	
+			<article class="content">
+				<img class="image" src="img/image.png">			
+				<div class="article">
```

Comment:
don't create tag article and class article which refers to different elements. Better rename